src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/BackupUtilities.cs

```diff
@@ -136,6 +147,109 @@ public BackupConfigInfo GetBackupConfigInfo(string databaseName)
         }
 
         /// <summary>
+        /// Execute backup
+        /// </summary>
+        public void PerformBackup()
+        {
+            this.backup = new Backup();
```

Comment:
>this.backup = new Backup(); [](start = 12, length = 27)

so we can only have 1 backup running at a time per instance of this class?  should have some logic to enforce this?  i.e. check if a backup is currently running and either exit the method or raise exception or abort operation..etc.